test/Microsoft.AspNet.Mvc.TestCommon/ResourceFile.cs

```diff
@@ -66,16 +66,14 @@ public static Stream GetResourceStream(Assembly assembly, string resourceName, b
             var stream = assembly.GetManifestResourceStream(fullName);
             if (sourceFile)
             {
-                // Normalize line endings to '\n' (LF). This removes core.autocrlf, core.eol, core.safecrlf, and
+                // Normalize line endings to '\r\n' (CRLF). This removes core.autocrlf, core.eol, core.safecrlf, and
                 // .gitattributes from the equation and treats "\r\n", "\r", and "\n" as equivalent. Does not handle
                 // some obscure line endings (e.g. "\n\r") but otherwise ensures checksums and line mappings are
                 // consistent.
                 string text;
                 using (var streamReader = new StreamReader(stream))
                 {
-                    text = streamReader.ReadToEnd()
-                        .Replace("\r\n", "\n")  // Windows line endings
-                        .Replace("\r", "\n");   // Older Mac OS line endings
+                    text = streamReader.ReadToEnd().Replace(Environment.NewLine, "\r\n");
```

Comment:
This will break on Windows machines with `core.autocrlf=false` in the `git` configuration. Need to be more general without changing `"\r\n"` to `"\r\n\n"`.